test/blas_test.hpp

```diff
@@ -178,6 +178,24 @@ class BLAS_Test<blas_test_args<ScalarT_, MetadataT_, ExecutorType_>>
     }
   }
 
+  template <typename DataType,
+            typename value_type = typename DataType::value_type>
+  static void set_const(DataType &vec, size_t _N, value_type c) {
+    for (size_t i = 0; i < _N; ++i) {
+      vec[i] = c;
+    }
+  }
+
+  template <typename DataType,
+            typename value_type = typename DataType::value_type>
+  static void set_monotonic(DataType &vec, size_t _N, value_type increment) {
+    value_type v = value_type(0);
+    for (size_t i = 0; i < _N; ++i) {
+      vec[i] = v;
+      v += increment;
+    }
```

Comment:
isn't this std::iota? why the need for a static method instead of using the algorithm?

Reply:
Yup - and it also turns out that this function isn't even used any more (it was useful for debugging), so I'll remove it.